teachertool/src/components/DebouncedInput.tsx

```diff
@@ -0,0 +1,28 @@
+import { useRef } from "react";
+import { Input, InputProps } from "react-common/components/controls/Input";
+
+export interface DebouncedInputProps extends InputProps {
+    intervalMs?: number | undefined;
```

Comment:
Consider assigning the default value here so it's less buried and available to intellisense.
```ts
    intervalMs = 500
```